roles/openshift_metrics/README.md

```diff
@@ -109,3 +109,73 @@ Author Information
 ------------------
 
 Jose David Martín (j.david.nieto@gmail.com)
+
+Image update procedure
+----------------------
+An upgrade of the metrics stack from older version to newer is an automated process and should be performed by calling appropriate ansible playbook and setting required ansible variables in your inventory as documented in https://docs.openshift.org/.
+
+Following text describes manual update of the metrics images without version upgrade. To determine the current version of images being used you can:
+```
+oc describe pod | grep 'Image ID:'
+```
+
+A way to determine when was your image last updated:
+```
+$ docker images
+REPOSITORY                                       TAG     IMAGE ID       CREATED             SIZE
+<registry>/openshift3/origin-metrics-cassandra   v3.7    9c3597aeb39f   2 weeks ago         780 MB
+
+$ docker inspect 9c3597aeb39f 
+[
+    {
+        . . .
+        "RepoDigests": [
+            "docker.io/openshift3/origin-metrics-cassandra@sha256:5a8b576dd4d66fa0e5a2a1e1805c7677eee6c208110ebe9d1a0587094914a7e3"
+        ],
+        . . .
+        "Config": {
```

Comment:
This example could also stand to show information such as the release to be more like the logging example above.

Reply:
thanks, updated. Btw our upstream images, for example `docker.io/openshift/origin-metrics-cassandra:latest`, do not provide that information, is it by design or would we like to label those images with `release`, `version`, and `url` as well?